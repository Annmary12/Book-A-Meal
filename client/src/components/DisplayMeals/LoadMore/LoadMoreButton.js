import React from 'react';
import PropTypes from 'prop-types';
import styles from './load-more.scss';

const LoadMore = ({ handleClick }) => (
  <div className={styles.container}>
    <button
      className={styles.loadMoreBtn}
      onClick={handleClick}
    >
      Load more
    </button>
  </div>
);

LoadMore.propTypes = {
  handleClick: PropTypes.func.isRequired
};

export default LoadMore;
