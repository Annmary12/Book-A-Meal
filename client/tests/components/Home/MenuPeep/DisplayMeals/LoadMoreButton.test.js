import React from 'react';
import LoadMoreButton from '../../../../../src/components/Home/MenuPeep/DisplayMeals/LoadMore/LoadMoreButton';

describe('LoadMoreButton component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<LoadMoreButton />);

    expect(wrapper).toMatchSnapshot();
  });
});