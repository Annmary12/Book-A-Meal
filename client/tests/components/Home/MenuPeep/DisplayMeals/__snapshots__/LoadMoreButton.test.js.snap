// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadMoreButton component should match snapshot 1`] = `
<div
  className="container"
>
  <button
    className="loadMoreBtn"
  >
    Load more
  </button>
</div>
`;
